src/build/_importlib.py

```diff
@@ -1,13 +1,8 @@
 import sys
 
 
-if sys.version_info < (3, 8):
+if sys.version_info < (3, 10, 2):
     import importlib_metadata as metadata
-elif sys.version_info < (3, 9, 10) or (3, 10, 0) <= sys.version_info < (3, 10, 2):
```

Comment:
Please don't remove the try/except part. That helps bootstrapping (where the `.json` property isn't required).

Reply:
I added the try/except and a comment with the bootstring. Do you think it's correct and understandable?